src/Pose.cpp

```diff
@@ -0,0 +1,16 @@
+#include <ZividPython/Matrix.h>
+#include <ZividPython/Pose.h>
+
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+namespace ZividPython
+{
+    void wrapClass(pybind11::class_<Zivid::HandEye::Pose> pyClass)
+    {
+        pyClass.def(py::init([](const Eigen::Matrix4d &matrix) {
```

Comment:
Change this line to: 
`pyClass.def(py::init([](const Eigen::Matrix<double, 4, 4, Eigen::RowMajor> &matrix) `